File: PerfTester-Xamarin/PerfTester-Xamarin/Screens/Home/HomeController.cs
using System;
using CoreFoundation;
using UIKit;
using System.Collections.Generic;

namespace PerfTesterXamarin.Screens.Home
{
    public partial class HomeController : UIViewController
    {
        public HomeController() : base("HomeController", null)
        {
            SetupNavigationBar();
        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
        }

        void SetupNavigationBar()
        {
            Title = "Home";
        }
    }
}

File: PerfTester-Xcode/PerfTester-Xcode/Screens/Home/HomeController.swift
//
//  HomeController.swift
//  PerfTester-Xcode
//
//  Created by Patryk Średziński on 08.06.2018.
//  Copyright © 2018 private.perftesterxcode. All rights reserved.
//

import UIKit

class HomeController: UIViewController {

    init() {
        super.init(nibName: nil, bundle: nil)
        setupNavBar()
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }

    private func setupNavBar() {
        self.title = "Home"
    }
}
